frontend/server/handlers/artifacts.ts

```diff
@@ -383,11 +383,18 @@ function getNamespaceFromUrl(path: string): string | undefined {
   return params.get('namespace') || undefined;
 }
 
+function isValidNameSpace(namespace: string) {
```

Comment:
Add a test in https://github.com/kubeflow/pipelines/blob/6a55449ffb5b3d935b32033dc9a13b1852c059a4/frontend/server/integration-tests/artifact-proxy.test.ts#L11 to validate this change.